.scrutinizer.yml

```diff
@@ -25,12 +25,8 @@ filter:
         - vendor/*
 
 tools:
-    external_code_coverage:
-        timeout: 3600
-        runs: 3 # Scrutinizer will wait for 3 code coverage submissions
```

Comment:
does scrutinizer still hang after removing this one?

Reply:
No it just does not expect coverage now... Just runs static analysis and PHP mess detector etc :) 
![image](https://user-images.githubusercontent.com/1352979/97798383-de11ac80-1c25-11eb-8ee5-9a51e0fb1989.png)